src/app/public/modules/text-editor/toolbar/text-editor-toolbar.component.ts

```diff
@@ -91,10 +95,29 @@ export class SkyTextEditorToolbarComponent implements OnInit {
   public set styleState(value: SkyTextEditorStyleState) {
     this._styleState = value;
     if (value.font !== this.styleStateFontName) {
-      this.styleStateFontName = this.getFontName(value.font);
+      if (value.font === '\"Blackbaud Sans\", Arial, sans-serif') {
+        this.styleStateFontName = this.getFontName('Blackbaud Sans');
+      } else {
+        this.styleStateFontName = this.getFontName(value.font);
+      }
+    }
+  }
+
+  @Input()
+  public set disabled(value: boolean) {
+    const coercedValue = SkyFormsUtility.coerceBooleanProperty(value);
+    if (coercedValue !== this.disabled) {
+      this._disabled = coercedValue;
+      this.changeDetector.markForCheck();
     }
   }
 
+  public get disabled() {
```

Comment:
Needs a return type.

Reply:
fixed in "[add return type](https://github.com/blackbaud/skyux-text-editor/pull/21/commits/0f02eff4dd8c95de71ff7cac711a9525485004df)"